.github/workflows/CI.yml

```diff
@@ -0,0 +1,113 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  pull_request:
+    branches:
+      - "master"
+      - "maintenance/.+"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.cfg.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          # Uncomment when ready ;)
+          # - 3.8
+        cfg:
```

Comment:
Is it possible to give each job in the matrix a more descriptive name, or have the env variables printed in this output? I like how travis makes it clear which is True and which is False

![image](https://user-images.githubusercontent.com/7935382/78148598-a413ca00-73fa-11ea-95df-6b5ea5d216bc.png)

![image](https://user-images.githubusercontent.com/7935382/78148682-c60d4c80-73fa-11ea-8596-6778394fc397.png)

Reply:
I don't think we have control over that autogenerated name :( Let me check.

Reply:
Oh, I stand corrected, I think we can use a [`name:`](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idname) label to override that. Let me push! This is a great addition!

Reply:
How does it look now? 😃

Reply:
Beautiful! Thanks you!